.github/workflows/update-helm-repo.yaml

```diff
@@ -175,6 +162,18 @@ jobs:
           fi
           echo "packagename=${name}-${version}" >> $GITHUB_OUTPUT
 
+      - name: Add dependency chart repos
+        run: |
+          cd source
+          # Skip the header line and make sure that tabs are expanded into spaces
+          deps=$(helm dependency list "${{ steps.parse-chart.outputs.chartpath }}" | tail +2 | expand)
+          while read -r row; do
+            IFS=' ' read -ra parts <<< "$row"
```

Comment:
Let's already account for OCI dependencies, if I add minio as OCI dependency in Mimir:
```
dependencies:
  - repository: oci://registry-1.docker.io/bitnamicharts
    name: minio
    version: "12.10.3"
```

I get this from helm dependency list:
```
$ helm dependency list
walk.go:74: found symbolic link in path: /home/krajo/go/github.com/grafana/mimir/operations/helm/charts/mimir-distributed/mixins/alerts.yaml resolves to /home/krajo/go/github.com/grafana/mimir/operations/mimir-mixin-compiled/alerts.yaml
walk.go:74: found symbolic link in path: /home/krajo/go/github.com/grafana/mimir/operations/helm/charts/mimir-distributed/mixins/dashboards resolves to /home/krajo/go/github.com/grafana/mimir/operations/mimir-mixin-compiled/dashboards
walk.go:74: found symbolic link in path: /home/krajo/go/github.com/grafana/mimir/operations/helm/charts/mimir-distributed/mixins/rules.yaml resolves to /home/krajo/go/github.com/grafana/mimir/operations/mimir-mixin-compiled/rules.yaml
NAME                  	VERSION	REPOSITORY                              	STATUS
minio                 	12.10.3	oci://registry-1.docker.io/bitnamicharts	ok    
grafana-agent-operator	0.3.21 	https://grafana.github.io/helm-charts   	ok    
rollout-operator      	0.15.0 	https://grafana.github.io/helm-charts   	ok 
```

And `helm repo add` fails for the oci one. So I'd do an if to ignore "oci://" repositories.